src/test/unit/lang/parser/user_defined_functions_test.cpp

```diff
@@ -39,6 +39,10 @@ TEST(parserFunctions, funsBad2) {
   test_throws("functions-bad2","Function declared, but not defined");
 }
 
+TEST(parserFunctions, funsBad2_good) {
+  is_parsable("src/test/test-models/bad/functions-bad2.stan",0, true);
+}
```

Comment:
I'm confused as to why a program in directory `bad` is being tested for parsability.  Although we're not enforcing cpplint in tests, we want spaces after commas in general for new code.